modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/java/com/liferay/frontend/taglib/clay/sample/web/internal/display/context/VerticalNavDisplayContext.java

```diff
@@ -69,6 +69,35 @@ public List<VerticalNavItem> getVerticalNavItems() {
 		return _verticalNavItems;
 	}
 
+	public List<VerticalNavItem> getVerticalNavSecurityItems() {
+		_verticalNavItems = new VerticalNavItemList() {
+			{
+				IntegerWrapper integerWrapper = new IntegerWrapper(1);
+
+				while (true) {
+					if (integerWrapper.getValue() == 2) {
+						break;
+					}
+
+					add(
+						verticalNavItem -> {
+							verticalNavItem.setHref(
+								"#" + integerWrapper.getValue());
+							verticalNavItem.setLabel(
+								"<script>document.body.style.backgroundColor=" +
+									"\"blue\"</script> ");
```

Comment:
We shouldn't be using plain <script> tags due to recent CSP restrictions. I'd just use a bit of different html to check against

Reply:
Hello @bryceosterhaus 
Would you mind linking me the CSP restrictions as well as what would be a better alternative, if the docs don;t specify?
Thank you for taking a look at this!

Reply:
So CSP means we can't have just bare <script> tags anymore. We either need to add them via an `aui:script` tag in a jsp or we need to add a "nonce" like this https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/run_scripts_in_element.es.js#L10-L12

However, since this is just a sample module, can you just add another bit of HTML to test instead of a script tag? Like a <b> or something that we can just visually see?

Reply:
Thank you for the help!
I made changes, let me know if anything else needs to be changed.